lib/astarte/client/appengine/devices.ex

```diff
@@ -57,6 +103,18 @@ defmodule Astarte.Client.AppEngine.Devices do
     end
   end
 
+  @doc """
+  Returns a values snapshot for a given interface without `:path` option, otherwise
```

Comment:
Returns a value ~~s~~ snapshot for a given interface without `:path` option, otherwise

`or`

Returns a snapshot **of values** for a given interface without `:path` option, otherwise